pkg/runtime/k0s/init.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k0s
+
+import (
+	"fmt"
+)
+
+func (k *Runtime) init() error {
+	pipeline := []func() error{
+		k.MergeConfigOnMaster0,
+		k.SSHKeyGenAndCopyIDToHosts,
```

Comment:
I'm not very familiar with k0s and I have a question:  is this necessary to configure ssh key login?